benefits/oauth/middleware.py

```diff
@@ -16,8 +22,19 @@ def process_request(self, request):
             # from the base middleware class, the session didn't have a verifier
             return result
 
-        if session.verifier(request).uses_auth_verification:
+        verifier = session.verifier(request)
+
+        if verifier.uses_auth_verification:
+            # all good, the chosen verifier is configured correctly
             return None
+        elif verifier.api_url is None and verifier.form_class is None:
```

Comment:
Not sure if it's only in my configuration, but if I don't set an Auth provider and leave both Eligibility API base URL and form_class blank, it looks like `verifier.api_url` and `verifier.form_class` evaluate to `""` and `""` doesn't evaluate to `None` in the condition so the new error page is not shown. If I change the condition to
```
elif verifier.api_url == "" and verifier.form_class == "":
```
then the condition does evaluate to true in this case and I see the new error page.

Reply:
Thanks @lalver1 -- I updated the conditional and squashed it into the latest commit [`5bce79d`](https://github.com/cal-itp/benefits/pull/2227/commits/5bce79d175967e3e382187fb959e961390552213)